config/config-builder/src/swarm_config.rs

```diff
@@ -154,6 +154,7 @@ impl SwarmConfig {
         );
         // NOTE: Need to restart upstream node with new configuration.
         let mut configs = Vec::new();
+        template.execution.genesis_file_location = PathBuf::from("genesis.blob");
```

Comment:
btw why do we generate the genesis tx and save it to `genesis.blob` instead of generating it and handling the bytes directly when we need it? (and never touching the filesystem)

Reply:
Why do we generate all the configs :(
If you look at vm/vm-genesis it generates a genesis that is only used for testing the vm. This could arguably be made as part of run time testing and make folks lives a little easier. I suspect one of the reasons is to help identify when we have broken compatibility, but I don't think that's the right way to do it.